apps/opik-backend/src/main/java/com/comet/opik/infrastructure/AppMetadataService.java

```diff
@@ -12,11 +11,10 @@ public interface AppMetadataService {
 
 @RequiredArgsConstructor(onConstructor_ = @Inject)
 class AppMetadataServiceImpl implements AppMetadataService {
-    @Config
-    private final MetadataConfig config;
+    private final OpikConfiguration config;
```

Comment:
Minor: annotate the field with `@NonNull`, so you'd notice about these DI errors.

Reply:
Done in https://github.com/comet-ml/opik/pull/513/commits/323e93246723f5dc5ff2d86d46d71cbe472de289